feature-libs/cart/base/core/http-interceptors/handlers/bad-cost-center-request.handler.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { HttpErrorResponse, HttpRequest } from '@angular/common/http';
+import { TestBed } from '@angular/core/testing';
+import {
+  GlobalMessageService, GlobalMessageType, HttpErrorHandler,
+  HttpResponseStatus,
+} from '@spartacus/core';
+import { BadCostCenterRequestHandler } from './bad-cost-center-request.handler';
+
+
+const MockRequest = {} as HttpRequest<any>;
+
+const MockRandomResponse = {} as HttpErrorResponse;
+
+const MockCostCenterErrorResponse = {
+  error: {
+    errors: [
+      {
+        type: 'EntityValidationError',
+        message: 'The application has encountered an error.',
+      },
+    ],
+  },
+  url: 'https://mockURL/occ/v2/powertools-spa/users/mockUsr/carts/00001004/costcenter?costCenterId=mockCostCtr',
+} as HttpErrorResponse;
+
+
+class MockGlobalMessageService {
+  add() {}
+  remove() {}
+}
+
+describe('BadCostCenterRequestHandler', () => {
+  let service: BadCostCenterRequestHandler;
+  let globalMessageService: GlobalMessageService;
```

Comment:
naming the `service` var to something more self explanatory would make the code more readable.  The new name could be as simple as `handler`.